classroom.rb

```diff
@@ -0,0 +1,14 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @student.push(student)
```

Comment:
- [ ] This should be `@students.push(student)` based on what you defined in the `initialize` method. Kindly fix that.

Reply:
Thank you for bringing my attention to this, the change has been implemented.